tests/spine-order-001/OPS/package.opf

```diff
@@ -1,26 +1,31 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
 <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
-  <dc:title>Spine Order 001</dc:title>
+  <dc:title>Package Document Spine Order 001</dc:title>
```

Comment:
There is only one test of this type, right? If so I think you should remove 001. Yes, there could be more tests in the future, but that's true of any, and my preference is to name them descriptively, like "Package Document Spine Order" and/or "Package Document Spine Order - Something Something".

See also https://github.com/w3c/epub-tests#naming (which is open for discussion, BTW).